libs/blocks/merch/merch.js

```diff
@@ -100,6 +100,14 @@ window.tacocat.loadPromise = new Promise((resolve) => {
 
   loadScript(literalScriptUrl)
     .catch(() => ({})) /* ignore if literals fail */
+    .then(() => {
+      // load custom elements polyfill for Safari only.
+      if (navigator.userAgent.indexOf('Safari') >= 0 && navigator.userAgent.indexOf('Chrome') < 0) {
+        const base = miloLibs || codeRoot;
+        return loadScript(`${base}/deps/custom-elements-1.3.0.js`);
```

Comment:
@overmyheadandbody good point. JQuery dep has also version suffix but other deps don't have it.
If the consensus is to not have it, I'll rename it.
@npeltier @vladen @3ch023 @narcis-radu  thoughts?

Reply:
@yesil I vote for simpler version-free filename

Reply:
I'd say it comes down to a case-to-case basis if you actually want/need different versions (for example for different blocks) but generally I don't think it makes a lot of sense to keep the version numbers

Reply:
I agree here and also don't see an immediate need for maintaining an explicit version.

Reply:
I prefer having the version number if it isn't in the source (which I don't see).  Otherwise we won't know if we need/want to upgrade it in the future?  Not a big deal either way though.

Reply:
We don't need for an explicit version so it makes sense to load the code with a file name that's more general. We would still have to manually upgrade and for this reason check the base version, but @overmyheadandbody will take care of it - just add the version number to the file.

Reply:
@chrischrischris 
I also think that adding the version in the dep file should be sufficient as @narcis-radu suggested.
Thus, when we update the file, from git history standpoint, it is still the same and we can trace changes back easily.